sympy/geometry/ellipse.py

```diff
@@ -323,6 +323,28 @@ def circumference(self):
         else:
             return 4 * self.major * elliptic_e(self.eccentricity**2)
 
+    def Ellipse_Circumference(self):
+        """
+        Computes the circumference of an ellipse using length of semi-axes as a and b.
+        Require a >= 0 and b >= 0. This is much faster than the circumference method given above.
+        Relative accuracy is about 0.5 ** 53.
```

Comment:
You should mention that it computes the result numerically, using double precision.